jxboot-resources/templates/700-jenkins-ing.yaml

```diff
@@ -4,8 +4,6 @@ kind: Ingress
 metadata:
   annotations:
     kubernetes.io/ingress.class: nginx
-    nginx.ingress.kubernetes.io/auth-secret: jx-basic-auth
```

Comment:
Does this not mean that access to Jenkins X is not un-authenticated?

Reply:
I think the installation already sets up the regular form based security in jenkins. So this basic-auth imho is redundant

Reply:
hmm, if the form-based login is still there I guess that would be fine. Unfortunately, I am not familiar with this area.